src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String CHAR_SEPARATOR = ",";
+
+    private File myFile = null;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        myFile = new File(fromFileName);
+        try (FileReader fileReader = new FileReader(myFile);
+                BufferedReader bufferedReader = new BufferedReader(fileReader)) {
+            String value = bufferedReader.readLine();
+            if (value == null) {
+                throw new RuntimeException("File is empty");
+            }
+            int supplySum = 0;
+            int buySum = 0;
+            String[] tempStr;
+            while (value != null) {
+                tempStr = value.split(CHAR_SEPARATOR);
+                if (tempStr[0].equals(SUPPLY)) {
+                    supplySum += Integer.parseInt(tempStr[1]);
+                } else if (tempStr[0].equals(BUY)) {
+                    buySum += Integer.parseInt(tempStr[1]);
+                } else {
+                    throw new RuntimeException("Error in csv file");
+                }
+                value = bufferedReader.readLine();
+            }
```

Comment:
Extract `read` operation in separate method, calculations can leave here or also extract to separate method. Don't use global variables to keep data, return as result of calling method.